### PR TITLE
replace links with our own urls by G1ANT.Manual.Builder

### PR DESCRIPTION
**Pull request by WiktoriaPrusik**
it is connected with: 
https://github.com/G1ANT-Robot/G1ANT.Language/pull/2
https://github.com/G1ANT-Robot/G1ANT.Addon/pull/108
https://github.com/G1ANT-Robot/G1ANT.Addon.Core/pull/5
https://github.com/G1ANT-Private/G1ANT.Website/pull/334
https://github.com/G1ANT-Robot/G1ANT.Tutorials/pull/9

### CODE REVIEW

File: g1ant.robot-window/auxiliary-windows/mouse-position.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ When you open this tool, you are prompted to choose absolute or relative measure
 
 ![](../../-assets/mouse-position.jpg)
 
-Right after clicking your answer, the screen grays out and you can point your mouse cursor to the desired position. When you click the left mouse button, the current mouse position will be inserted into your code as a value of [`⟦point⟧`](G1ANT.Language/G1ANT.Language/Structures/PointStructure.md) structure.
+Right after clicking your answer, the screen grays out and you can point your mouse cursor to the desired position. When you click the left mouse button, the current mouse position will be inserted into your code as a value of [`⟦point⟧`](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/PointStructure.md) structure.
```
Comment (MarekKosinski) [inline g1ant.robot-window/auxiliary-windows/mouse-position.md:9]: Link does not work.

File: appendices/common-arguments.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ If a given color is not found, a message “*no expected color in this position*
 
 This argument allows assigning a variable, which will store the information about the exception that occurred when a command was executed. The variable name should be preceded by `♥`.
 
-The error information is contained in an [error structure](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) with `type` and `message` indexes.
+The error information is contained in an [error structure](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) with `type` and `message` indexes.
```
Comment (MarekKosinski) [inline appendices/common-arguments.md:111]: Link does not work.

File: g1ant.robot-window/auxiliary-windows/settings.md [hunk 3]
```diff
@@ -8,12 +8,12 @@ The Settings window is a front-end to G1ANT.Robot.config XML file. Its path is d
...
-| **Triggers**    | This is the place for setting up new [triggers](../../g1ant-language/triggering/readme.md) or modifying existing ones. To add a new trigger, right-click **Triggers** item and choose `Add`. To delete a particular trigger, right-click it and choose `Remove`. |
-| **Variables**   | Here you can declare global [variables](../../g1ant-language/variables/creating-new-variables.md) to be used by all scripts run by this G1ANT.Robot. To add a new trigger, right-click **Variables** item and choose `Add`. To delete a particular variable, right-click it and choose `Remove`. |
+| **Triggers**    | This is the place for setting up new [triggers](https://manual.g1ant.com/link/G1ANT.Manual/g1ant-language/triggering/readme.md) or modifying existing ones. To add a new trigger, right-click **Triggers** item and choose `Add`. To delete a particular trigger, right-click it and choose `Remove`. |
```
Comment (MarekKosinski) [inline g1ant.robot-window/auxiliary-windows/settings.md:17]: Link does not work.

File: g1ant.robot-window/auxiliary-windows/settings.md [hunk 4]
```diff
@@ -8,12 +8,12 @@ The Settings window is a front-end to G1ANT.Robot.config XML file. Its path is d
...
+| **Variables**   | Here you can declare global [variables](https://manual.g1ant.com/link/G1ANT.Manual/g1ant-language/variables/creating-new-variables.md) to be used by all scripts run by this G1ANT.Robot. To add a new trigger, right-click **Variables** item and choose `Add`. To delete a particular variable, right-click it and choose `Remove`. |
```
Comment (MarekKosinski) [inline g1ant.robot-window/auxiliary-windows/settings.md:18]: Does not work.

**Review by MarekKosinski — APPROVED** [review]
I approve although 4 of the changed links does not work - Wiktoria said on slack that it is Jacek's fault and will be fixed by him.

File: appendices/special-variables.md [hunk 5]
```diff
@@ -6,44 +6,44 @@ For a full description of a variable and examples of its usage, click its name.
...
-| [`♥timeoutwindow`](G1ANT.Addon.Core/Variables/TimeoutWindowVariable.md) | integer | Determines the timeout value for the [window](G1ANT.Language/G1ANT.Addon.Core/Commands/WindowCommand.md) command; the default value is 1000 |
+| [`♥timeoutchrome`](https://manual.g1ant.com/link/G1ANT.Addon.Core/Variables/TimeoutChromeVariable.md) | integer | Determines the timeout value for the [chrome](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Addon.Core/Commands/ChromeCommand.md) command; the default value is 10000 |
+| [`♥timeoutconnect`](https://manual.g1ant.com/link/G1ANT.Addon.Core/Variables/TimeoutConnectVariable.md) | integer | Determines the timeout value for the [is.accessible](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/develop/G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/IsAccessibleCommand.md) and [ping](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/develop/G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/PingCommand.md) commands; the default value is 1000 |
```
Comment (Vegebutcher) [inline appendices/special-variables.md]:
```suggestion
| [`♥timeoutconnect`](https://manual.g1ant.com/link/G1ANT.Addon.Core/Variables/TimeoutConnectVariable.md) | integer | Determines the timeout value for the [is.accessible](https://manual.g1ant.com/link/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/IsAccessibleCommand.md) and [ping](https://manual.g1ant.com/link/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/PingCommand.md) commands; the default value is 1000 |
```

File: appendices/special-variables.md [hunk 6]
```diff
@@ -6,44 +6,44 @@ For a full description of a variable and examples of its usage, click its name.
...
+| [`♥timeoutremotedesktop`](https://manual.g1ant.com/link/G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/TimeoutRemoteDesktopVariable.md) | integer | Determines the timeout value for the [vnc.connect](https://manual.g1ant.com/link/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/VncConnectCommand.md) command; the default value is 10000 |
+| [`♥timeoutrest`](https://manual.g1ant.com/link/G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/TimeoutRestVariable.md) | integer | Determines the timeout value for the [rest](https://manual.g1ant.com/link/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/RestCommand.md) command; the default value is 5000 |
+| [`♥timeoutwaitforcolor`](https://manual.g1ant.com/link/G1ANT.Addon.Core/Variables/TimeoutWaitForColorVariable.md) | integer | Determines the timeout value for the [waitfor.color](https://github.com/G1ANT-Robot/G1ANT.Addon.Core/blob/develop/G1ANT.Addon.Core/Variables/G1ANT.Language/G1ANT.Addon.Core/Commands/WaitforColorCommand.cs) command; the default value is 10000 |
```
Comment (Vegebutcher) [inline appendices/special-variables.md]:
```suggestion
| [`♥timeoutwaitforcolor`](https://manual.g1ant.com/link/G1ANT.Addon.Core/Variables/TimeoutWaitForColorVariable.md) | integer | Determines the timeout value for the [waitfor.color](https://manual.g1ant.com/link/G1ANT.Language/G1ANT.Addon.Core/Commands/WaitforColorCommand.cs) command; the default value is 10000 |
```

File: getting-started/support.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
 # Support
 
-Should you require any technical support, please contact our Support Team [via email](mailto:support@g1ant.com) or by [filling a form](https://g1ant.fogbugz.com/default.asp?pg=pgPublicEdit), and one of our team will assist with your query as soon as possible.
+Should you require any technical support, please contact our Support Team [via email](https://manual.g1ant.com/link/G1ANT.Manual/getting-started/mailto:support@g1ant.com) or by [filling a form](https://g1ant.fogbugz.com/default.asp?pg=pgPublicEdit), and one of our team will assist with your query as soon as possible.
```
Comment (Vegebutcher) [inline getting-started/support.md:3]: exception for mailto:

File: appendices/special-variables.md (hunk 5 above)
Comment (Vegebutcher) [inline appendices/special-variables.md]:
```suggestion
| [`♥timeoutconnect`](https://manual.g1ant.com/link/G1ANT.Addon.Core/Variables/TimeoutConnectVariable.md) | integer | Determines the timeout value for the [is.accessible](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/develop/G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/IsAccessibleCommand.md) and [ping](https://manual.g1ant.com/link/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/IsAccessibleCommand.md) and [ping](https://manual.g1ant.com/link/G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/PingCommand.md) commands; the default value is 1000 |
```

**Review by Vegebutcher — APPROVED** [review]

**Pull request merged**